client/components/Update/UpdateSidebar.vue

```diff
@@ -59,15 +60,39 @@ export default {
               this.$router.push({
                 params,
                 name: userViewName,
-              });
+              }).catch(()=>{});
               this.$store.commit('setUserFilter', email);
             },
             active: this.$route.name === userViewName
               && this.$store.state.userFilter === email,
             children: [],
           })),
         },
-
+        {
+          name: 'Tags',
+          onClick: () => {
+            // this.$router.push({
```

Comment:
Do we want similar behavior to the all "User" tab? Where no filter just means to display all updates under each tag? It might be confusing for the user if all other header tabs are clickable but this one

Reply:
I was thinking that it would be confusing because one update can have multiple tags and in that case would we just have duplicates? A todo for this would be to make it visually unclickable

Reply:
Oh hmmm that is indeed true. I guess either works as long as its obvious its unclickable